content/news/2023-02-20-global-team-recruiting/index.en.md

```diff
@@ -0,0 +1,104 @@
+
+Some of the personal benefits that we think come with joining the global team and volunteering time are gaining experience in leadership, asynchronous remote work, international team work, not-for-profit open-source work, and work in a women(+) only team and organization.
+
+Depending on the role, there also may be opportunities for technical upskilling.
+For example, some members of the Global team use R, reproducible reports, git/GitHub, GitHub Actions, Hugo website development, and database integrations.
+(Knowledge of these tools is not a requirement for selection; the Global team will provide training on processes specific to your role.)
+
+# Commitment
+
+We ask those seeking consideration to commit to one year of 1 to 4 hours a week for service to the Global Team.
+Of course, vacations and exceptions may occur.
+
+# Positions
+
+Below is a list of all current global team operations. All teams are listed, however
+not all teams are currently recruiting. Please refer to this table when submitting
+your interest. 
+
+In addition, applying to a specific team does not mean your contributions are limited to that
+team. Individual teams often solicit and receive feedback or assistance from other
+members of the Global Team.
+
+
+|Team|Brief responsibilities|Seeking # members|
+|:----|:----|:----|
+|Social Media|Post regularly on social media platforms (e.g., Twitter, Mastodon, LinkedIn, etc.) and create a regular newsletter|3|
+|Website|Maintenance / further development of the website. Review potential translations coming in, review json files added / updated.|2|
+|Blog administration|Recruit, review and support guest bloggers. Guests are welcome to cross-post to their own blogs.|2|
+|Chapter Mentoring|Help organizers create or reactivate a chapter. Match mentees to their mentors and follow-up mentorships.|2|
+|IWD Campaign|A new team for the International Women's Day Campaign. Create, disseminate, and engage in communications.|2|
+|YouTube|Respond to requests to add a video or managers to the global YouTube channel.|1|
+|CoC enforcement|Receive, review a solve CoC reports. Periodically revise CoC and process. Support organizers and moderators.|1|
+|Chapter on-boarding|Share information on how to create a new chapter, invite prospective organizers to the R-Ladies Community Slack, and add chapter to internal records.|1|
+|Directory management|Review and approve new R-Ladies Global directory entries to be posted on the website.|1|
+|Meetup Pro|Create new chapters on Meetup.|1|
+|Zoom + calendly|Monitor the Zoom + Calendly integration for booking zoom for meetups. Help and support organizers book and set up their zoom.|1|
+|RUGS grant administration| |0|
+|Abstract Review System| |0|
+|RoCur Twitter Account|Nominate, onboard, and support curators.|0|
+|E-mail management| |0|
```

Comment:
should we add some sort of categorization? For instance email management is similar to Zoom+calendly (mostly admin); other roles are more political etc.

Reply:
are we sure we dont need anyone on email?

@maelle do you think further categorisation might make it clearer what the roles are?

Reply:
I was thinking more about adding a categorisation of members that often interact/coordinate with eachother.

Reply:
We need someone on email!!!

Reply:
@yabellini  i changed email to managment to 1. is that sufficient? i also need a description fora few of these teams, including email management.

Reply:
i like the idea of categorization but i would need suggestions on how to organize it.